### PR TITLE
[ipa-resign] Migrated to Node10

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: ipa-resign

**Description**: Migrated task to Node 10.

**Documentation changes required:** No

**Merge after this PR is complete:**  https://github.com/microsoft/build-task-team/issues/676

**Added unit tests:** No

**Attached related issue:** [#670](https://github.com/microsoft/build-task-team/issues/670)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Review by anatolybolshakov — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
Let's make buddy testing before meting of this PR

File: Tasks/ipa-resign/task.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "demands": [ "xcode" ],
     "version": {
         "Major": "1",
-        "Minor": "178",
-        "Patch": "2"
+        "Minor": "182",
```
Comment (anatolybolshakov) [inline Tasks/ipa-resign/task.json:15]: Please bump version to current sprint number

File: Tasks/IpaResign/Tests/L0.ts [hunk 2]
```diff
@@ -20,7 +20,7 @@ describe('ipa-resign L0 Suite', function () {
     });
     /* tslint:enable:no-empty */
```
Comment (DaniilShmelev) [inline Tasks/IpaResign/Tests/L0.ts:22]: Could you please also fix test timeout here [like this](https://github.com/microsoft/app-store-vsts-extension/pull/204/commits/cc0c4bc1e3f9c5fff3649f781bc4a01e009175d6)? This task doesn't have any tests right now, but we may run into an issue with timeouts if we decide to add them later

Other than that, LGTM!

**Review by DaniilShmelev — APPROVED** [review]

**Comment by egor-bryzgalov** [comment]
Tested the task manually. Found no issues.

File: Tasks/ipa-resign/task.json (hunk 1 above)
Comment (egor-bryzgalov) [reply Tasks/ipa-resign/task.json]: Thanks! Resolved.

File: Tasks/IpaResign/Tests/L0.ts (hunk 2 above)
Comment (DaniilShmelev) [reply Tasks/IpaResign/Tests/L0.ts:22]: Fixed separately in https://github.com/microsoft/app-store-vsts-extension/pull/212

**Pull request merged**